sandbox/testing_impl-gda_examples.xml

```diff
@@ -0,0 +1,50 @@
+<gda:procedure xlink:href="wxt520" xlink:title="wxt520"/>
```

Comment:
I'm OK with merging this now, but for `0.4` this XML file should be used for proper tests, not just in the sandbox.

Reply:
See https://github.com/52North/sos4R/issues/74